TeX_files/chapter08.tex

```diff
@@ -158,9 +158,9 @@ \subsection*{Procedure:}
 \end{enumerate}
 
 
-If the load impedance is given and we are asked to find the reflection coefficient and load impedance at another point "l" on the transmission line that is:
+If the load impedance is given and we are asked to find the reflection coefficient and load impedance at another point $l$ on the transmission line, that is:
 
-Given $Z_L$ or $Y_L$, if we are to find $Z_L$ or $Y_L$ at distance l from the load, we use the hypothetical sketch of the smith chart shown in figure~\ref{fig:uytrewsxcvbj}
+Given $Z_L$ or $Y_L$, if we are to find $Z_L$ or $Y_L$ at distance $l$ from the load, we use the hypothetical sketch of the smith chart shown in figure~\ref{fig:uytrewsxcvbj} below;
```

Comment:
The figure isn't always positioned where it is intended because the arrangement is handled by the compiler such that the best position is chosen.
```suggestion
Given $Z_L$ or $Y_L$, if we are to find $Z_L$ or $Y_L$ at distance $l$ from the load, we use the hypothetical sketch of the smith chart shown in figure~\ref{fig:uytrewsxcvbj};
```